public/mockServiceWorker.js

```diff
@@ -0,0 +1,303 @@
+/* eslint-disable */
```

Comment:
Auto generated.

Reply:
This being in the public folder, will this be included in the output bundle?

Reply:
It's not included in the output of the TypeScript compiler or the UMD bundle created by esbuild. It is included in the Storybook build.